builder/commands/update.py

```diff
@@ -38,17 +40,19 @@ def take_action(self, parsed_args):
             print parser.print_help()
             sys.exit(1)
         config = self.parse_config(parsed_args.conf)
-        yaml_file = os.path.join(parsed_args.yaml)
-
-        if os.path.isdir(yaml_file):
-            views = [view for view in os.listdir(yaml_file)]
-            for view_yaml in views:
-                self.log.debug("View file %s" % view_yaml)
-                self.read_update(config,
-                                 os.path.join(yaml_file + '/%s' % view_yaml))
-        else:
-            self.log.debug("View file %s" % yaml_file)
-            self.read_update(config, os.path.join(yaml_file))
+        for yaml_filename in parsed_args.yaml:
+
+            yaml_file = os.path.join(yaml_filename)
+
+            if os.path.isdir(yaml_file):
+                views = [view for view in os.listdir(yaml_file)]
```

Comment:
This could lead to some confusion for the end user. We have two mechanisms of providing multiple view yamls now. User can either have multiple views in a folder and give that as an argument or can specify multiple view yamls on the command line itself or a combination of both.

I am not necessarily against this idea, just trying to think it through. Let me know your thoughts as well.